docs/getting_started/using_modin.rst

```diff
@@ -173,16 +174,10 @@ Specifying ``num_cpus`` limits the number of processors that Modin uses. You may
 specify more processors than you have available on your machine, however this will not
 improve the performance (and might end up hurting the performance of the system).
 
-Examples
---------
-You can find an example on our recent `blog post`_ or on the `Jupyter Notebook`_ that we
-used to create the blog post.
-
 .. _`DataFrame`: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html
 .. _`pandas`: https://pandas.pydata.org/pandas-docs/stable/
 .. _`open an issue`: https://github.com/modin-project/modin/issues
 .. _`autoscaler documentation`: https://ray.readthedocs.io/en/latest/autoscaling.html
 .. _`Ray's documentation`: https://ray.readthedocs.io/en/latest/api.html
-.. _`blog post`: https://rise.cs.berkeley.edu/blog/pandas-on-ray-early-lessons/
-.. _`Jupyter Notebook`: https://gist.github.com/devin-petersohn/f424d9fb5579a96507c709a36d487f24#file-pandas_on_ray_blog_post_0-ipynb
 .. _`tools and features`: comparisons/upcoming_features.html
+.. _sklearn LinearRegression: https://github.com/modin-project/modin/blob/master/examples/modin-scikit-learn-example.ipynb
```

Comment:
where is the link to this entry?

Reply:
Moved over the Examples section to use this entry.